### DISCUSSION TITLE
Is there a way to get the index of clicked page?

### DISCUSSION THREAD

**Discussion by spyrosbazios** [hrynko/vue-pdf-embed#143] [Q&A]
I am trying to get the index of each page I click on (or is currently in view). I checked the example on README.md but so far I haven’t been successful. 
I modified slightly an existing jsfidle to do what I want but is in pure js using pdf js. (you can find it [here](https://jsfiddle.net/baziosspyros/6c1vswkf/9/))

Any assistance would be greatly appreciated! ❤️

**Comment by hrynko** [comment]
Hi @spyrosbazios,

There are currently no page-specific click handlers. But from the provided JSFiddle, it looks like it's not about clicks, but rather about the scroll event. Right?

**Reply by spyrosbazios** [reply]
Yes no matter how (click / hover or scroll event), I want to find a way to get the page index, when all the pages are shown and can be scrolled. (not when displaying the pages one by one)

Thank you very much for your fast response!

**Reply by hrynko** [reply]
Please check this reply https://github.com/hrynko/vue-pdf-embed/discussions/139#discussioncomment-5752888

Hope it helps
